contracts/HATVaults.sol

```diff
@@ -270,20 +284,24 @@ contract  HATVaults is Governable, HATMaster {
    * @dev dismiss a pending claim - called either by hats govenrance, or by anyone if the claim is over 5 weeks old
    * @param _pid pool id
   */
-    function dismissPendingApprovalClaim(uint256 _pid) external {
+    function dismissPendingApprovalClaim(uint256 _pid) external onlyArbitrator {
         // solhint-disable-next-line not-rely-on-time
-        require(msg.sender == governance() || pendingApprovals[_pid].createdAt + 5 weeks < block.timestamp, "HVE09");
+        // NOTE: The timeout check is done on arbitrator's side.
```

Comment:
So I'd uncomment the line you removed here

Reply:
> ... we still need to signal to the connector that the claim has been prematurely dismissed, otherwise it'll try to enforce the ruling of the belated dispute which might cause unexpected behaviour.

Why do you need to "signal the connector"? Can the connector not check if the claim is dismissed before trying to enforce the ruling? 

Perhaps identifying claims by the poolId instead of having a unique id to refer to a claim is making this a bit more complex then it should be - an dperhaps we should consider making that change, wdyt?

Reply:
Exactly. Since the claim is identified by poolID it makes it difficult to check. Let's say the claim was dismissed by HATVaults and is not active anymore, so if the dispute ended at this exact time it would check and not enforce it. But what if it's resolved a bit later when new claim with this pID was created? Then the dispute thinks it's an old claim and enforces the ruling for the incorrect dispute

Reply:
Yes, but this is a problem in HATVaults, and I do not think we should concessions here in the integration because of that. So we will add claimIds #200